packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -99,16 +109,14 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     event LiquidityAdded(address indexed token, uint256 amount, uint256 lpTokensMinted, address liquidityProvider);
     event LiquidityRemoved(address indexed token, uint256 amount, uint256 lpTokensBurnt, address liquidityProvider);
     event DepositRelayed(
-        uint32 indexed relayId,
+        bytes32 indexed depositHash,
         DepositData depositData,
-        address slowRelayer,
         address l1Token,
-        uint64 realizedLpFeePct,
```

Comment:
hhmm. not having this could be an issue. it makes the off-chain bot work a bit harder. what exactly is the flow now the bot must go through te get this data to validate it in the price feed in the OO dispute bot.

Reply:
nvm this is baked into the `RelayData`!

Reply:
![image](https://user-images.githubusercontent.com/9457025/136238237-eaee676a-72c6-467f-8fd9-abe9a4551c9d.png)